Formula/udp2raw-multiplatform.rb

```diff
@@ -0,0 +1,30 @@
+class Udp2rawMultiplatform < Formula
+  desc "Multi-platform(cross-platform) version of udp2raw-tunnel client"
+  homepage "https://github.com/wangyu-/udp2raw-multiplatform"
+  url "https://github.com/wangyu-/udp2raw-multiplatform/archive/refs/tags/20210111.0.tar.gz"
+  sha256 "712ad3c79b6ef5bf106c615823d0b0b3865d1c957f9838cf05c23b7ac7024438"
+  license "MIT"
+
+  depends_on "make" => :build
+  depends_on "libnet"
+  depends_on :macos
+
+  def install
+    ENV["OPT"] = (system_command "libnet-config", args: ["--cflags"]).stdout
+    system "make", "mac"
+    bin.install "udp2raw_mp"
+    etc.install "example.conf" => "udp2raw_client.conf"
```

Comment:
Is there no `make install`?

Reply:
There is no make install to do this.

Reply:
What is there a `make install` for?

Reply:
The hand-written makefile do not support make install.